src/components/article-body/body-components/container.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentFactory } from '../component-factory';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+export const Container = ({
+    argument,
+    children,
+    ...rest
+}: {
+    argument: any;
+    children: any;
+}) => {
+    return (
+        <div>
```

Comment:
Could we use a fragment here instead of a `div`?

```
<>
            {children.map((element: ArticleNode, index: number) => (
                <ComponentFactory {...rest} nodeData={element} key={index} />
            ))}
        </>
```